.circleci/config.yml

```diff
@@ -1614,7 +1614,7 @@ jobs:
             set -eu
             sudo apt-get install jq
             PIPELINE_ID=$(curl 'https://circleci.com/api/v2/project/gh/DataDog/dd-trace-php/pipeline?branch=PHP-5' | jq -r '.items[0].id')
-            WORKFLOW_ID=$(curl "https://circleci.com/api/v2/pipeline/$PIPELINE_ID/workflow" | jq -r '.items[]|select(.name == "build_packages").id')
+            WORKFLOW_ID=$(curl "https://circleci.com/api/v2/pipeline/$PIPELINE_ID/workflow" | jq -r '.items[]|select(.name == "build_packages").id' | head -1)
```

Comment:
Sorry, what does this fix in practice? I understand you are limiting it to one line, but what was causing multiple lines in the first place?

Reply:
That happens when you re-run a workflow.